### DISCUSSION TITLE
issue preventing me from submitting my github student pack request

### DISCUSSION THREAD

**Discussion by ghost** [community/community#36590] [GitHub Education] [answered]
Hello,
I allow myself to write to you because I am currently encountering a problem that prevents me from submitting my request for github student pack, let me explain. I am currently in a school called Lyon Ynov Campus, I registered on github with the school's email address, and then I go to the 'Get your GitHub benefits' page.

First of all what shocks me is that the email address of my account is not detected (screen-01). 
![screen-01](https://user-images.githubusercontent.com/116114412/196527777-169340af-7d9d-451e-9e02-6b68afa0982f.png)

I click on 'add and verify your school-issued email address in your account settings', this one is added and verified (screen-02)
![screen-02](https://user-images.githubusercontent.com/116114412/196527808-55187eaf-19ca-451e-9c8b-76f98cd79e5a.png)

Even if no address is filled in and I can't fill in one, I try to enter the name of the school 'Lyon Ynov Campus' anyway (unknown by github), while taking into account that you must fill in as indicated the field how you intend to use your github account.

I then get an error: 'Please select the email address you use for the school - ideally the one issued by your school.' (screen-03 / screen-04)
![screen 03](https://user-images.githubusercontent.com/116114412/196527882-dd64d081-c479-4c46-ade1-98d0820213e9.png)
![screen-04](https://user-images.githubusercontent.com/116114412/196527889-c6d017de-f210-4714-bef7-5df1cabf529c.png)

The global preview of my page is in the screen-05 (I am well connected but my email is not associated)
![Screen-05](https://user-images.githubusercontent.com/116114412/196527947-b3c3be33-1138-4b5f-a747-bcaffefebedd.png)

I turn around...

Already last year I had encountered this problem and my school had been unable to help me so I abandoned GITHUB. I would love to get into it!

Please help me!!

**Comment by haseeb-xd** [comment, marked as answer]
`If you don’t have a school-issued email, follow the prompts to fill out additional information.You’re still eligible even if you only have a personal Gmail address, as long as you can provide alternative documentation to verify your current student status.`

Error is occurring because you are leaving "how do you plan to use github" field empty. Write a reason in that field, and you are good to go

**Comment by ghost** [comment]
Hello @haseeb-xd,

First of all thank you for your answer and your help, and I apologize for the polite formulas that I have not been respected.

There was something new and I managed to enter the email address! But I now have a new problem during validation that says:

> Your school is a GitHub Campus partner. Please contact your school administrator to have the correct benefits applied to your GitHub account.

![2022-10-20 21_07_57-Request a discount - GitHub Education](https://user-images.githubusercontent.com/116114412/197037766-1be4b483-fde9-4248-a455-ab3f6153fcb8.png)


My school tells me that they can't do anything on their side. I don't know if it's in the school to do some things or on the github side...

thank you in advance for the participation and good evening to you
